apps/web/src/components/Badge/Badge.tsx

```diff
@@ -57,20 +61,17 @@ export default function Badge({ badgeRef, eventContext }: Props) {
   }, []);
 
   const isEnabled = useMemo(() => {
-    return BADGE_ENABLED_COMMUNITY_ADDRESSES.has(contract?.contractAddress?.address ?? '');
-  }, [contract?.contractAddress?.address]);
+    return (
+      BADGE_ENABLED_COMMUNITY_ADDRESSES.has(contract?.contractAddress?.address ?? '') || imageURL
+    );
```

Comment:
hmm problem with this logic is that if the contract address is disabled but it at least has an `imageURL`, the existence of the imageURL will override and display the badge

<img width="242" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/4b1275da-5425-4729-a9f3-97c56be9989c">

in the above example, we don't want to show the crypto coven badge because the contract isn't a part of the enabled community addresses.

will fix this logic and push up a change!

Reply:
Thanks for the push @Robinnnnn! Totally missed that we have crypto coven badge too